src/logging/LoggedComand.java

```diff
@@ -0,0 +1,32 @@
+package logging;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class LoggedComand extends Command {
+
+    public static boolean IS_LOG = false;
+
+    private String commandName = getClass().toString();
```

Comment:
You dont have to specify `command`, because the class name contains the command.

For example, for class `Person`, we dont specify `personName`, we say `name`